src/main/java/org/example/services/UserInteractionService.java

```diff
@@ -0,0 +1,31 @@
+package org.example.services;
+
+import java.util.Scanner;
+import org.example.exceptions.InvalidArgument;
+import org.example.prompt.MenuPrompt;
+import org.example.validators.UserInteractionServiceValidator;
+
+public class UserInteractionService implements UserInteractionServiceInterface {
+  private int userChoice;
+
+  Scanner scanner = new Scanner(System.in);
```

Comment:
This code is not formatted, use cmd + opt + shift + L

Reply:
formatted code